kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -329,22 +402,23 @@ def audio_analyser_callback(self, order):
         # this boolean will notify the main process that the order have been processed
         self.order_analyser_return = True
 
-    def get_no_voice_flag_from_request(self, http_request):
+    def get_boolean_flag_from_request(self, http_request, json_name):
```

Comment:
Instead of having a method to get only boolean ? can we have a method which can find a flag not depending from the type ?
eg: get_flag_from_request()

Reply:
here the problem only appear when we send a boolean through the api. the word False is interpreted like True.